pages/index.js

```diff
@@ -1,21 +1,72 @@
 import Card from '../components/Card';
 
+const techies = [
+  {
+    id: 1,
+    firstName: 'Rochelle',
+    lastName: 'Rossman',
+    imageUrl:
+      'https://avatars.githubusercontent.com/u/102299152?s=400&u=2fab77da66f2df4181579defc0bc881109d553c0&v=4',
+    title: 'Software Engineer',
+    linkedInUrl: 'https://www.linkedin.com/in/rochelle-rossman/',
+    githubUrl: 'https://github.com/rochelle-rossman',
+    portfolioUrl: 'https://rochellerossman.dev',
+    resumeUrl:
+      'https://docs.google.com/document/d/1h1L2PI_raWf3lXm0VepStg76ioLKc8jqX9dzmxbLjfM/edit',
+    jobType: 'Software Engineer',
+    experienceLevel: 'Junior',
+  },
+  {
+    id: 2,
+    firstName: 'Ewell',
+    lastName: 'Van der Brug',
+    imageUrl: 'http://dummyimage.com/176x100.png/cc0000/ffffff',
```

Comment:
@rochelle-rossman also, we are getting build errors on these. I know they are place holders, but let's use https: so the build will run